test/manager/bundler/artifacts.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('bundler.updateArtifacts()', () => {
     platform.getFile.mockResolvedValueOnce('Current Gemfile.lock');
     fs.outputFile.mockResolvedValueOnce(null as never);
     exec.mockImplementationOnce((cmd, _options, callback) => {
-      callback(null, '', '');
+      callback(null, { stdout: '', stderr: '' });
```

Comment:
Suprisingly, that's the signature which returns correct `ExecResult` in the code.